packages/types/src/protocol.ts

```diff
@@ -20,8 +20,8 @@ export interface AppCaptureProtocolInterface {
   afterSpec (): void
   beforeTest(test: Record<string, any>): void
   afterTest(test: Record<string, any>): void
-  commandLogAdded (log: any, timestamp: string): void
```

Comment:
any reason we don't type the `log` in the protocol signature?

Reply:
Hopefully we'll be able to share the types from the other side so we can correctly type these. Right now it would just be double work and since this is just a pass-through, it doesn't seem worthwhile.